### PR TITLE
提交鸿蒙arkts语法整改代码

### PR DESCRIPTION
**Pull request by zhaoyang57**
提交鸿蒙arkts语法整改代码

### CODE REVIEW

File: templates/js-template-default/frameworks/runtime-src/proj.openharmony/entry/src/main/ets/cocos/game.ts [hunk 1]
```diff
@@ -20,13 +21,13 @@ function boot() {
...
     }
     cc.view.enableAutoFullScreen([
-      cc.sys.BROWSER_TYPE_BAIDU,
-      cc.sys.BROWSER_TYPE_BAIDU_APP,
-      cc.sys.BROWSER_TYPE_WECHAT,
-      cc.sys.BROWSER_TYPE_MOBILE_QQ,
-      cc.sys.BROWSER_TYPE_MIUI,
-      cc.sys.BROWSER_TYPE_HUAWEI,
-      cc.sys.BROWSER_TYPE_UC,
+    cc.sys.BROWSER_TYPE_BAIDU,
```
Comment (qiuguohua) [inline templates/js-template-default/frameworks/runtime-src/proj.openharmony/entry/src/main/ets/cocos/game.ts]: Should indentation be required here?

File: templates/js-template-default/frameworks/runtime-src/proj.openharmony/entry/src/main/ets/common/PortProxy.ts [hunk 2]
```diff
@@ -21,57 +21,56 @@
...
+import { ThreadWorkerGlobalScope } from '@ohos.worker';
 import { MessageEvent } from '@ohos.worker';
 
 export class PortProxy {
-    private autoId: number = 0;
-    public actionHandleMap = {}
-    private port :DedicatedWorkerGlobalScope = null;
+  private autoId: number = 0;
+  public actionHandleMap = {}
+  private workerPort: ThreadWorkerGlobalScope = null;
```
Comment (qiuguohua) [inline templates/js-template-default/frameworks/runtime-src/proj.openharmony/entry/src/main/ets/common/PortProxy.ts]: Why do we need to name it ‘workerPort’ when this interface accepts UIPort and workerPort?

File: templates/js-template-default/frameworks/runtime-src/proj.openharmony/entry/src/main/ets/components/CocosWebView.ets [hunk 3]
```diff
@@ -63,33 +67,33 @@ export class WebViewInfo {
...
       })
-      // 开启DOM存储权限
-      .domStorageAccess(true)
-      // 开启数据库存储权限
-      .databaseAccess(true)
-      // 图片加载相关权限
-      .imageAccess(true)
-      // 支持JS代码运行
-      .javaScriptAccess(true)
+      .domStorageAccess(true)// 开启DOM存储权限
```
Comment (qiuguohua) [inline templates/js-template-default/frameworks/runtime-src/proj.openharmony/entry/src/main/ets/components/CocosWebView.ets]: Use of english comments？

File: templates/js-template-link/frameworks/runtime-src/proj.openharmony/entry/src/main/ets/workers/cocos_worker.ts [hunk 4]
```diff
@@ -22,90 +22,82 @@ import hilog from '@ohos.hilog';
...
+      nativeWebView.shouldStartLoading(msg.param.viewTag, msg.param.url);
+      break;
+    case "onPageEnd":
+      nativeWebView.finishLoading(msg.param.viewTag, msg.param.url);
+      break;
+    case "onErrorReceive":
+      nativeWebView.failLoading(msg.param.viewTag, msg.param.url);
+      break;
+    case "onVideoEvent":
+    // @ts-ignore
```
Comment (qiuguohua) [inline templates/js-template-link/frameworks/runtime-src/proj.openharmony/entry/src/main/ets/workers/cocos_worker.ts]: extra space

File: templates/js-template-link/frameworks/runtime-src/proj.openharmony/entry/src/main/ets/workers/cocos_worker.ts [hunk 5]
```diff
@@ -22,90 +22,82 @@ import hilog from '@ohos.hilog';
...
+      const renderContext = cocos.getContext(ContextType.NATIVE_RENDER_API);
+      renderContext.nativeEngineInit();
+      launchEngine().then(() => {
+        console.info('launch CC engine finished');
+      }).catch(e => {
+        console.error('launch CC engine failed');
+      });
+    // @ts-ignore
+      globalThis.oh.postMessage = nativeContext.postMessage;
+    // @ts-ignore
```
Comment (qiuguohua) [inline templates/js-template-link/frameworks/runtime-src/proj.openharmony/entry/src/main/ets/workers/cocos_worker.ts]: extra space

**Review by qiuguohua — APPROVED** [review]

**Review by minggo — APPROVED** [review]

**Pull request merged**